taskana-adapter-camunda-tasklistener/src/main/java/pro/taskana/adapter/camunda/schemacreator/TaskanaOutboxSchemaCreator.java

```diff
@@ -0,0 +1,92 @@
+package pro.taskana.adapter.camunda.schemacreator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import javax.sql.DataSource;
+
+import org.apache.ibatis.jdbc.ScriptRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class TaskanaOutboxSchemaCreator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaOutboxSchemaCreator.class);
+    private static final String SQL = "/sql";
+    private static final String DB_SCHEMA = SQL + "/taskana-outbox-schema.sql";
+    private static final String DB_SCHEMA_DB2 = SQL + "/taskana-outbox-schema-db2.sql";
+    private static final String DB_SCHEMA_POSTGRES = SQL + "/taskana-outbox-schema-postgres.sql";
+    private static final String DB_SCHEMA_DETECTION = SQL + "/taskana-outbox-schema-detection.sql";
+    private static final String DB_SCHEMA_DETECTION_POSTGRES = SQL + "/taskana-outbox-schema-detection-postgres.sql";
+
```

Comment:
files taskana-outbox-schema.sql, taskana-outbox-schema-detection.sql and taskana-outbox-schema-detection-postgres.sql are missing in project taskana-adapter-camunda-tasklistener.
I don't think we should provide them in this pull request, but we should open 
- a ticket to provide schema version support for the outbox db table
- another ticket to make the schema name configurable for the outbox

Reply:
deleted the Strings for schema detection for now